test/resources/examples/todo/src/service/TasksRouteService.tsx

```diff
@@ -1,46 +1,47 @@
 
-import { ITasksRouteService } from '../rest/ITasksRouteService';
-import { Task } from "../entities/Task";
+import type { Request, Response } from 'express';
+import { TasksRoutes } from 'generated:rest/TasksRoutes';
+import { Task } from "generated:entities/Task";
 
-export class TasksRouteService implements ITasksRouteService {
+export class TasksRouteService implements TasksRoutes {
 
 
         /**
         * Add task for user
-        * HTTP: POST /tasks/{userId}/{id}
+        * HTTP: POST /tasks/:userId/:id
         */
-        addTask( userId:string, id:string, task:Task, overwrite:boolean ): Promise<void> {
-            throw new Error('REST resource method not implemented: "addTask"');
+        addTask( req:Request<{'userId': string, 'id': string},void, Task, void>, res:Response<void> ): void {
```

Comment:
Looks like prettier didn't run on the outputted file?

`{'userId': string, 'id': string}` should be formatted as
`{userId: string, id: string}`

Reply:
It is being passed through prettier - not sure why it doesn't catch it

Reply:
Found it - we were missing tsx files